Fix xdr mirroring issue (#34581)
demisto/content: Packs/CortexXDR/ReleaseNotes/6_1_41.md

#### Integrations

##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Fixed an issue where updating the incident status in XDR when mirrored to XSOAR would result in a "resolved_other" status if resolved.
> - Fixed an issue where duplicate incident statuses would not change the alert status.(?) 

Reviewer: ```suggestion
- Fixed an issue in mirroring where updating the incident status in XDR, would incorrectly set the status to "resolved_other" if the incident was resolved.
```
